Description: Adds import warnings in conf.py
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-06-06

--- python-osprofiler-1.3.0.orig/doc/source/conf.py
+++ python-osprofiler-1.3.0/doc/source/conf.py
@@ -16,6 +16,7 @@ import os
 import subprocess
 import sys
 import time
+import warnings
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
